### PR TITLE
Added RP website to the page

### LINKED ISSUE #104: RP page doesn't have the link to RP website
see title

### PR DESCRIPTION
**Pull request by traugertyson**
Attempting to fix #104

### CODE REVIEW

**Comment by aashishkapur** [comment]
Do we want to include the 2017 website on the [list of RP websites](https://github.com/acm-uiuc/groot-desktop-frontend/blob/master/server.js#L90)?

File: app/views/desktop/conference.ejs [hunk 1]
```diff
@@ -17,7 +17,7 @@ this license in a file with the distribution.
 <div class="row">
   <p>
     Reflections | Projections, founded in 1995, is a technology conference operated by students in ACM@UIUC. Striving to cater to our community of students, alumni, and professionals, we provide a forum to share and learn about current advancements in computer science and its applications. We have tech talks from across the industry and academia, workshops and puzzle-solving events for attendees, and Mechmania, our annual artificial intelligence programming competition. The entire conference is completely free for anyone to attend, regardless of age, major, or affiliation with UIUC.
-    This year, Reflections | Projections will occur September 29th - October 1st, immediately following ACM Career Week, which features a job fair, a startup fair and company visits.
+    This year, Reflections | Projections will occur September 29th - October 1st, immediately following ACM Career Week, which features a job fair, a startup fair and company visits. For more information, please vist the website: <a href = "http://reflectionsprojections.com/">http://reflectionsprojections.com/</a>
```
Comment (tyler-thetyrant) [inline app/views/desktop/conference.ejs]: vist

**Comment by narendasan** [comment]
Yeah add them to the list and map the conference and conference/2017 urls to reflectionsprojections.org

**Comment by tyler-thetyrant** [comment]
@narendasan They haven't added .org domain to CNAME. 

cc @bluedrops

**Comment by narendasan** [comment]
any update on this?

**Comment by aashishkapur** [comment]
@traugertyson Can you add the 2017 website [here](https://github.com/acm-uiuc/groot-desktop-frontend/blob/master/server.js#L89)?

After that we should be good to go

**Comment by narendasan** [comment]
can someone also make sure we are using .org as the primary url for RP? .com for a conference is super weird

**Comment by narendasan** [comment]
not sure who to ask about that

**Comment by pranaygp** [comment]
Just linking to the discussion we had [here](https://github.com/ReflectionsProjections/rp2017/pull/24)

**Comment by narendasan** [comment]
@traugertyson can you change the url you listed to reflectionsprojections.org instead of .com? both in the text and in the server?

**Comment by narendasan** [comment]
@traugertyson thanks for sticking with this, couple things I want to verify before merge. @aashishkapur is there a 2017 specific url setup yet? I'd rather have the server point to /conference/2017 than the main url so we dont need to change it every year. @pranaygp @aashishkapur @sskhandek  is https not enabled on the RP site yet?

**Comment by aashishkapur** [comment]
I'll set up the /conference/2017 redirect pointing to the main website. 

Regarding https, I'm setting up a vm for RP this week, so it'll be set up once we transfer over from GitHub pages.

**Comment by narendasan** [comment]
If you can manage it in a way where you can have:
```
/conference/2017 - - - - - - - - - > server
                                                 
reflectionsprojections.org - - - - > server                                               
```
or 
```
reflectionsprojections.org - - -> /conference/2017 - - -> server
```
over 
```
 /conference/2017 - - -> reflectionsprojections.org - - - >server
````
That would be ideal

**Comment by aashishkapur** [comment]
I was planning to do the last one.

**Comment by narendasan** [comment]
I guess that's fine if we can host the site after rp on github pages or something

**Comment by narendasan** [comment]
Might make it easier future to just redirect the domain to the right site instead of recofiguring things around

File: server.js [hunk 2]
```diff
@@ -88,6 +88,7 @@ app.get('/conference', function(req, res) {
   res.render('desktop/conference', {
     authenticated: utils.isAuthenticated(req),
     editions: [
+      { year: '2017', path: 'http://reflectionsprojections.org/' },
```
Comment (aashishkapur) [inline server.js]: @traugertyson Can you change the path here to 'https://www-s.acm.illinois.edu/conference/2017' ?

**Pull request merged**